test/g_searcher/search_results_test.exs

```diff
@@ -26,7 +26,7 @@ defmodule GSearcher.SearchResultsTest do
 
   describe "get_search_result/1" do
     test "returns {:ok, search_result} given a valid ID" do
-      search_result = insert(:search_result)
+      search_result = insert(:search_result) |> forget(:search_result_urls)
```

Comment:
How about we define a search term with a unique value when fabricating a record and use it in the assertion? So we can reduce complexity from additional function (`forget/2`). I think don't need to assert the whole record, a unique value is in the attribute is enough.

What do you think?

Reply:
I wasn't really a fan of making unique values because if we want it thorough, it would need to match each field here and have more assertions. It made the test feel needlessly long. 😂 

I've updated the helper to forget all the associations instead so we don't have to keep track of what field to forget in the test, what do you think?